pkg/api/types.go

```diff
@@ -1366,23 +1366,35 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty"`
 }
 
+// DaemonEndpoint contains information about a single Daemon endpoint.
+type DaemonEndpoint struct {
+	// Port number of the given endpoint.
+	Port int `json:port`
+}
+
+// NodeDaemonEndpoints lists ports opened by daemons running on the Node.
+type NodeDaemonEndpoints struct {
+	// Endpoint on which Kubelet is listening.
+	KubeletEndpoint DaemonEndpoint `json:"kubeletEndpoint,omitempty"`
+}
+
 // NodeSystemInfo is a set of ids/uuids to uniquely identify the node.
 type NodeSystemInfo struct {
-	// MachineID is the machine-id reported by the node
+	// machineID reported by the node.
```

Comment:
I meant either just english:
Machine ID reported by the node.

Or json field name + english:
machineID is the machine ID reported by the node.

For these fields, the latter is not so useful.

Sadly, we don't have a consistent convention, but we should never be referencing Go types/fields in the user-facing documentation.